lib/passport-configurator.js

```diff
@@ -507,6 +525,11 @@ PassportConfigurator.prototype.configureProvider = function(name, options) {
       }
       );
       break;
+    case 'custom':
+      let verifyMethod = options.verifyMethod || function() {};
+      delete options.verifyMethod;
```

Comment:
Is it necessary?

Reply:
Yes, other strategies have the verification method built into the passport-configurator.js file. This allows custom strategies to do what ever someone wants. In my use case I am currently using the verifyMethod to create the local account (for caching) to match the profile returned by the passport authentication process.

Reply:
@DOSarrest I was referring to `delete options.verifyMethod;`

Reply:
@raymondfeng The verifyMethod is only used within that portion of the code and is not used anywhere else. It also is bound to the PassportConfigurator so there is no use for it past that point. Considering it is the callback for AuthStrategy, there is no reason to pass it as an option to AuthStrategy.